LayoutTestBase/Config/LYTConfig.h

```diff
@@ -71,6 +71,15 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL accessibilityTestsEnabled;
 
 /**
+ If on, a snapshot of the view for the current failing test will be saved to the derived data folder. The data for the failing test is also logged in the index.html file for each LYTLayoutTestCase sub class. The path to the folder is logged after the test suite has finished.
+ 
+ This property can also be turned on/off on each LayoutTestCase.
+ 
+ Default: true
+ */
+@property (nonatomic) BOOL enableFailingTestSnapshots;
```

Comment:
Can we rename to match style of the other properties.

failingTestSnapshotsEnabled

also, maybe even:

testFailureSnapshotsEnabled?